gamemode/modules/jobs/sv_jobs.lua

```diff
@@ -156,7 +156,21 @@ function meta:changeTeam(t, force, suppressNotification, ignoreMaxMembers)
         gamemode.Call("PlayerSetModel", self)
         gamemode.Call("PlayerLoadout", self)
     else
-        self:KillSilent()
+        if GAMEMODE.Config.instantjob then
+            local vPoint = self:GetShootPos() + Vector(0,0,50)
+            local effectdata = EffectData()
+            effectdata:SetEntity(self)
+            effectdata:SetStart(vPoint)
+            effectdata:SetOrigin(vPoint)
+            effectdata:SetScale(1)
+            util.Effect("entity_remove", effectdata)
```

Comment:
This code duplicates the effect code of the `norespawn` if-branch. Could you unify the duplicate code?